### ISSUE TITLE
Consider routing over piers to get access to ferries

### ISSUE THREAD

**Issue by rkflx** [abrensch/brouter#263]
If BRouter routed over piers, it would be far easier to use ferries when planning a route. Currently you have to force the route by using a lot of via points (which have to be placed very precisely for them to work, if they help at all), and in some cases routing fails completely (see other issue).

[Test case](https://brouter.de/brouter-web/#map=18/50.95608/14.03367/standard&lonlats=14.031333,50.955913;14.034385,50.956474;14.034932,50.955997;14.033516,50.95494)

Routing in BRouter-Web (notice missing route segments):
![1](https://user-images.githubusercontent.com/27236544/88487837-e7391080-cf88-11ea-86d7-b50ff2894f16.png)

The [pier is already mapped](https://www.openstreetmap.org/way/457086565#map=19/50.95576/14.03518), though:
![3](https://user-images.githubusercontent.com/27236544/88487842-f15b0f00-cf88-11ea-9376-3a5af663c766.png)

**Comment by abrensch** [comment]
I see. Seems there were some edit wars on the issue when iD tried to enforce validation rules to add a highway tag in such cases: https://help.openstreetmap.org/questions/69818/should-a-pier-be-tagged-as-highway

Same for railway=platform : https://github.com/openstreetmap/iD/issues/6409

So maybe we should add both as implying highway=footway?

**Comment by rkflx** [comment]
Sounds reasonable.

**Comment by 0709wiwiwi** [comment]
> So maybe we should add both as implying highway=footway?   Yes pse.

Piers and Ferriess.

Brouter.
https://t.ly/kgf_

GraphHopper.
https://t.ly/mshB

OSRM.
https://t.ly/vYw1

Openstreetmap.
https://t.ly/Jduj

**Comment by polyscias** [comment]
Yes, I think it would be good to add [man_made=pier](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dpier) to lookups.dat, the wiki indicates it is a _raised walkway over water_, see also #416 

A problem is maybe that there are many more man_made tags (in ways context) that are not interesting, like storage_tank, cutline, pipeline, silo:

| Tag(s) | Count |
| --- | ---: |
| man_made=pier | 538425 ways |
| man_made=_\<other>_ | 2278655 ways |

You do not want these other's to be included

**Comment by afischerdev** [comment]
Added 
```
man_made=pier
public_transport=platform
```
to the #416
